_pytest/unittest.py

```diff
@@ -157,9 +156,20 @@ def runtest(self):
             self._testcase(result=self)
         else:
             # disables tearDown and cleanups for post mortem debugging (see #1890)
+            # but still implements the skipping machinery (see #2137)
+            testMethod = getattr(self._testcase, self._testcase._testMethodName)
+            if (getattr(self._testcase.__class__, "__unittest_skip__", False) or
+                getattr(testMethod, "__unittest_skip__", False)):
+                # If the class or method was skipped.
+                skip_why = (getattr(self._testcase.__class__, '__unittest_skip_why__', '') or
+                            getattr(testMethod, '__unittest_skip_why__', ''))
+                try:
```

Comment:
that was intentionally, as the python3 `_addSkip` has the same signature as the unittest2 `_addSkip` on python2, see [here](https://hg.python.org/unittest2/file/tip/unittest2/case.py#l518).

However, it was implicit. I explicitly added an comment and secured the PY2 path in 176c680.